examples/stable-diffusion/README.md

```diff
@@ -1,4 +1,4 @@
-<!---
```

Comment:
This shouldn't be removed as it is a comment in the README so that it doesn't appear when it is rendered

Reply:
fixed